playwright-typescript-playwright-test-main/playwright-typescript-playwright-test-main/lib/BaseTest.ts

```diff
@@ -1,7 +1,7 @@
-import { test as baseTest } from '@playwright/test';
-import { LoginPage } from '@pages/LoginPage';
+/* import { test as baseTest } from '@playwright/test';
```

Comment:
Remove unwanted and commented code

Reply:
We are not using this file so ,I removed this file